spec/index.html

```diff
@@ -76,26 +76,26 @@
 <body>
 <section id="abstract">
   <p>The Resource Description Framework (RDF) is a framework for
-    representing information in the Web.
+    representing information on the Web.
     This document defines an abstract syntax (a data model)
     which serves to link all RDF-based languages and specifications.
     The abstract syntax has two key data structures:</p>
   <ul>
     <li>RDF graphs are sets of subject-predicate-object triples,
-      where the elements may be IRIs, blank nodes, or datatyped literals.
+      where the elements may be IRIs, blank nodes, datatyped literals, or triple-terms.
       They are used to express descriptions of resources.</li>
     <li>RDF datasets are used to organize collections of RDF graphs,
-      are comprised of a default graph and zero or more named graphs.</li>
+      and consist of a default graph and zero or more named graphs.</li>
   </ul>
 
-  <p>RDF 1.2 introduces the ability to use an <a>RDF triple</a>
+  <p>Compared to RDF 1.1, RDF 1.2 introduces the ability to use an <a>RDF triple</a>
     as a <a>triple term</a>, in the <a>object</a> position of another <a>triple</a>.
     RDF 1.2 also introduces <a>directional language-tagged strings</a>,
     which contain a <a>base direction</a> element that allows the
     initial text direction to be specified for presentation by a user agent.</p>
 
-  <p>RDF 1.2 Concepts introduces key concepts and terminology for RDF 1.2, discusses
-    datatyping, and the handling of <a>fragment identifiers</a> in IRIs within
+  <p>This specification introduces key concepts and terminology for RDF 1.2, then discusses
```

Comment:
```suggestion
  <p>This specification introduces key concepts and terminology for RDF 1.2 and, thereafter, discusses
```

Reply:
I defer to the native spealers for that one :) @afs? @gkellogg?

Reply:
Riffing on @hartig's suggestion...
```suggestion
  <p>This specification introduces key concepts and terminology for RDF 1.2, and subsequently discusses
```